docs/detections/rules-ui-create.asciidoc

```diff
@@ -358,6 +360,30 @@ FROM logs-* [metadata _id, _index, _version]
 | LIMIT 10
 -----
 
+[float]
+[[esql-query-design]]
+===== Query design considerations 
+
+When writing your query, consider the following:
+
+- The {ref}/esql-commands.html#esql-limit[`LIMIT`] command specifies the number of rows an {esql} query returns and the number of alerts created per rule execution. Similarly, the <<opt-fields-all,`max_signals`>> setting specifies how many alerts a rule can create every time it runs.
++ 
+If the `LIMIT` value is lower than the `max_signals` value, the rule uses the `LIMIT` value. If the `LIMIT` is higher than `max_signals`, the rule uses `max_signals`. 
```

Comment:
```suggestion
If the `LIMIT` value is lower than the `max_signals` value, the rule uses the `LIMIT` value. If the `LIMIT` is higher than `max_signals`, the rule uses `max_signals` to determine the max number of generated alerts. 
```

Reply:
I'm not sure we need to be this explicit. These sentences are directly after a brief explanation of the `LIMIT` command and `max_signals` setting, so they're presented within that context. If you still think the text you added needs to be included for clarity, what do you think about adding it to the first sentence and adding "value" to the second sentence? For example:

```suggestion
If the `LIMIT` value is lower than the `max_signals` value, the rule uses the `LIMIT` value to determine the maximum number of alerts the rule generates. If the `LIMIT` value is higher than `max_signals` value, the rule uses `max_signals` value. 
```

Reply:
I like the last suggestion, let's go with this one